packages/kit/src/views/Send/pages/SendConfirm/TxAdvancedSettingsContainer.tsx

```
@@ -0,0 +1,336 @@
+            })}
+            name="nonce"
+            rules={{
+              validate: handleValidateNonce,
+              onChange: (e: { target: { name: string; value: string } }) => {
+                const value = e.target?.value;
+                let finalValue = '';
+
+                if (value === '') {
+                  finalValue = '';
+                } else {
+                  const formattedValue = parseInt(value, 10);
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)